<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main</class>
 <widget class="QWidget" name="main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>842</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Motor de Búsqueda</string>
  </property>
  <widget class="QLineEdit" name="lineQuerie">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>180</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>160</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Temas de interés:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonBuscar">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>230</y>
     <width>91</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Buscar</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>270</y>
     <width>531</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>90</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>JSON de BD:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineJSON">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>110</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>